### PR TITLE
Fpga spi speedup

### PR DESCRIPTION
**Pull request by mhtwn**
SPI speedup related changes.

### CODE REVIEW

File: src/main/scala/shell/xilinx/ArtyShell.scala [hunk 1]
```diff
@@ -170,7 +170,7 @@ abstract class ArtyShell(implicit val p: Parameters) extends RawModule {
         dut.qspi(0),
         dut.clock,
         dut.reset,
-        syncStages = qspi_params.sampleDelay
+        syncStages = 0
```
Comment (mwachs5) [inline src/main/scala/shell/xilinx/ArtyShell.scala]: why did you remove the syncstages? I would have thought you would have set this to syncStages = qspi_params.defaultSampleDel

Comment (mhtwn) [reply src/main/scala/shell/xilinx/ArtyShell.scala]: Agree. This was an old change, I needed to revert this to the default setting, as was done in other config files.

File: xilinx/vc707/vsrc/sdio.v [hunk 2]
```diff
@@ -17,8 +17,8 @@ module sdio_spi_bridge (
 );
 
   wire mosi, miso;
-  reg miso_sync [1:0];
-
+  //reg miso_sync [1:0];
```
Comment (mwachs5) [inline xilinx/vc707/vsrc/sdio.v]: can you delete this commented-out line, since you have reg miso_sync below here anyway?

File: xilinx/vc707/constraints/vc707-master.xdc [hunk 3]
```diff
@@ -1,3 +1,6 @@
 #-------------- MCS Generation ----------------------
 set_property CFGBVS GND                               [current_design]
 set_property CONFIG_VOLTAGE 1.8                       [current_design]
+
+set_property EXTRACT_ENABLE YES                       [get_cells dut_/spi_0_1/mac/phy/txd_reg*]
```
Comment (mwachs5) [inline xilinx/vc707/constraints/vc707-master.xdc]: what does this do? do you need this if you keep the synchronizer above ?

File: src/main/scala/shell/xilinx/ArtyShell.scala (hunk 1 above)
Comment (mhtwn) [reply src/main/scala/shell/xilinx/ArtyShell.scala]: updated.

File: xilinx/vc707/vsrc/sdio.v (hunk 2 above)
Comment (mhtwn) [reply xilinx/vc707/vsrc/sdio.v]: Updated.

File: xilinx/vc707/constraints/vc707-master.xdc (hunk 3 above)
Comment (mhtwn) [reply xilinx/vc707/constraints/vc707-master.xdc]: This is independent of the sync stages - there was a problem that we ( I and Henry S.)  saw with respect to the reset signal on the txd reg, hence we explicitly ensured it is pulled out.
